### ISSUE TITLE
feature request: jump to definition for dynamic lists

### ISSUE THREAD

**Issue by saidelike** [AndreasArvidsson/andreas-talon-vscode#9]
```python
@mod.capture(rule="{user.code_symbol}")
def code_symbol(m) -> str:
    return m.code_symbol
```

from `{user.code_symbol}` to `@ctx.dynamic_list("user.code_symbol")`:

```python
@ctx.dynamic_list("user.code_symbol")
def code_symbol_list(phrase) -> dict[str, str]:
    global spoken_map
    t = time.perf_counter()
    types = get_types_from_workspaces()
    spoken_map = generate_spoken_forms(types)
    print("Generating code_symbol list: ", len(spoken_map))
    print(f"{int((time.perf_counter()-t)*1000)}ms")
    return spoken_map
```

**Comment by AndreasArvidsson** [comment]
The extension doesn't really have anything to do with Talon lists. What exactly would the target for the extension be?

**Comment by AndreasArvidsson** [comment]
This is implemented in version `3.48.0`
